templates/web/index.md.erb

```diff
@@ -30,6 +30,61 @@ __NOTE:__ You will also need to include <code>require 'origen_jtag'</code> somew
 
 ### How To Use
 
+#### New Style Example
+
+The driver no longer requires specific pin names (or aliases), supports sub_block instantiation and DUTs with multiple JTAG ports.
+ You are no longer required to include "OrigenJTAG" in your DUT class.
+
+Here is an example integration:
+
+~~~ruby
+class Pioneer
+
+  include Origen::TopLevel
+
+  # TCK covers 4 tester cycles, 2 high then 2 low for each effective TCK pulse
+  # Strobe TDO only when TCK high.  Only store TDO on last cycle (3)
+  JTAG_CONFIG = {
+    :tclk_format => :rl,
+    :tclk_multiple => 4,
+    :tdo_strobe => :tclk_high,
+    :tdo_store_cycle => 3,
+  }
```

Comment:
I really don't like these constant class-level config options, they feel like a hangover since before we added the sub_block API.
In the multi-JTAG example here, does this provide default options which can be overridden in the individual sub_block definitions?

Reply:
I left that part alone.  The behavior is still the same.  The defaults can be overridden using that class config hash.  But, the overridden defaults still get overridden again by any options passed during instantiation.

I can change the doc example to be the expected use (pass the instantiation options in the sub_block call)

Reply:
Yeah, please, I think it would be good for this new example to really show that per-JTAG configuration is possible.